paris/src/main/java/com/airbnb/paris/typed_array_wrappers/TypedArrayTypedArrayWrapper.kt

```diff
@@ -47,6 +51,19 @@ internal class TypedArrayTypedArrayWrapper constructor(
     override fun getFraction(index: Int, base: Int, pbase: Int): Float =
         typedArray.getFraction(index, base, pbase, -1f)
 
+    override fun getFont(index: Int): Typeface? {
+        return if (isNull(index)) {
+            null
+        } else {
+            val resourceId = getResourceId(index)
```

Comment:
We could call `typedArray.getResourceId(index, 0)` directly here, the only other thing `getResourceId(index)` does is a null check which is redundant here.